test/csdl2markdown.test.js

```diff
@@ -96,7 +96,7 @@ describe("Non-OASIS Vocabularies", function () {
       "",
       "Term|Type|Description",
       ":---|:---|:----------",
-      'Foo *(Deprecated)*|[Tag](https://github.com/oasis-tcs/odata-vocabularies/blob/master/vocabularies/Org.OData.Core.V1.md#Tag)|<a name="Foo"></a>Use term `Foo` from the OASIS Core vocabulary instead',
+      'Foo *(Deprecated)*|[Tag](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.html#Tag)|<a name="Foo"></a>Use term `Foo` from the OASIS Core vocabulary instead',
```

Comment:
The problem with the HTML version is that links to the XML source code position do not work, this only works from Markdown files viewed via the GitHub UI.